tests/plugins/test_add_content_sets.py

```diff
@@ -61,66 +61,67 @@ def mock_content_sets_config(tmpdir, empty=False):
     tmpdir.join('content_sets.yml').write(yaml.safe_dump(content_dict))
 
 
-@pytest.mark.parametrize('meta_file_exists', [True, False])
+@pytest.mark.parametrize('manifest_file_exists', [True, False])
 @pytest.mark.parametrize('content_sets', [True, False])
 @pytest.mark.parametrize('platform', ['x86_64', 'ppc64', 's390x'])
-@pytest.mark.parametrize(('df_content, expected_df, base_layers, meta_file'), [
+@pytest.mark.parametrize(('df_content, expected_df, base_layers, manifest_file'), [
     (
         dedent("""\
             FROM base_image
             CMD build /spam/eggs
-            LABEL some=40
+            LABEL name=eggs version=1.0 release=42
         """),
         dedent("""\
             FROM base_image
             CMD build /spam/eggs
-            ADD metadata_2.json /root/buildinfo/metadata_2.json
-            LABEL some=40
+            ADD content_manifest.eggs-1.0-42.json /root/buildinfo/content_manifests/eggs-1.0-42.json
```

Comment:
We want `/root/content_manifests/eggs-1.0-42.json` not `/root/buildinfo/content_manifests/eggs-1.0-42.json`, right?

Reply:
No. This is the right location as per the design doc.